content/en/Getting started/what-to-expect.md

```diff
@@ -55,7 +55,7 @@ results. Here's what you can expect:
 1. We keep the Slack channel open until you've set each finding to:
    - Accepted Risk
    - Fixed
-   If you need access to the archived Slack channel, contact your Customer Success Manager or support@cobalt.io.
+      - {{% contact-csm-support %}} if you need access to the archived Slack channel.
```

Comment:
```suggestion

  {{% contact-csm-support %}} if you need access to the archived Slack channel.

```

I think this looks better (from my build)
![Screen Shot 2022-08-30 at 7 28 18 AM](https://user-images.githubusercontent.com/84352037/187463987-5c0c4c24-90ca-4a09-92d3-05b17c87f3b7.png)

Reply:
(Yes, I had messed up the formatting earlier.)